demo/ARContainerViewController.swift

```diff
@@ -6,19 +6,62 @@ import UIKit
 
 private let inset: CGFloat = 18.0
 
+private enum ARFeature {
+    case lane
+    case fence
+    case laneAndFence
+
+    var next: ARFeature {
+        switch self {
+        case .lane:
+            return .fence
+        case .fence:
+            return .laneAndFence
+        case .laneAndFence:
+            return .lane
+        }
+    }
+
+    var containsLane: Bool {
+        return self == .lane || self == .laneAndFence
+    }
+
+    var containsFence: Bool {
+        return self == .fence || self == .laneAndFence
+    }
+}
+
 final class ARContainerViewController: UIViewController {
     private lazy var mapViewController = ARMapNavigationController()
     private lazy var arViewController = VisionARViewController()
 
     weak var navigationDelegate: NavigationDelegate?
     private var navigationService: NavigationService?
+    private var activeARFeature: ARFeature = .lane {
```

Comment:
Do we want to have a case to disable both lane and fence (make them invisible)?

Reply:
no, as I got from the issue